net/Cargo.toml

```diff
@@ -6,14 +6,14 @@ edition = "2018"
 workspace = ".."
 
 [dependencies]
-actix = { version = "0.8.3", default-features = false }
+actix = { version = "0.11.0-beta.1", default-features = false }
```

Comment:
is safu use beta?

Reply:
We could merge this PR into a different branch and keep testing it until the final actix 0.11 release, while we update more components to the latest versions.